twilight-gateway/Cargo.toml

```diff
@@ -20,7 +20,7 @@ rand = { default-features = false, features = ["std", "std_rng"], version = "0.8
 serde = { default-features = false, features = ["derive"], version = "1" }
 serde_json = { default-features = false, features = ["std"], version = "1" }
 tokio = { default-features = false, features = ["net", "rt", "sync", "time"], version = "1.8" }
-tokio-tungstenite = { default-features = false, features = ["connect"], version = "0.18" }
+tokio-tungstenite = { default-features = false, features = ["connect"], git = "https://github.com/snapview/tokio-tungstenite.git" }
```

Comment:
Is depending on the git repository of `tokio-tungstenite` a good idea? Any breaking push might break Twilight. And Cargo doesn't allow publishing a crate if it depends on git repositories.

Reply:
Hence why it's a draft PR